ff_derive/Cargo.toml

```diff
@@ -15,6 +15,8 @@ edition = "2021"
 [features]
 # enabled when generating bitvec code utilizing the version of ff's bitvec
 bits = []
+# enabled when generating zeroize code utilizing the version of ff's zeroize
```

Comment:
For the `bits` feature flag, we need to synchronise the `bitvec` version with `ff` because it is used in the `ff` traits. Here, we have no such constraint, so we can just allow the user to provide whatever version of `zeroize` they want (as long as it has `DefaultIsZeroes`).
```suggestion
# enabled when generating zeroize bindings
```

Reply:
I was planning on adding `Zeroize` as a `cfg`-dependent trait bound in a follow up PR (I believe you previously mentioned "code generation changes" and "trait changes" should be in separate PRs).

Reply:
See https://github.com/Daniel-Aaron-Bloom/ff/commit/485adc5baac9fb550cef51b34fd0b67a2d3411ee for those changes

Reply:
Hmm, that change would not be accepted. Rust feature flags must be additive / expand the API, but adding a bound to a trait is a restricting change. The problem is that someone could be using `ff` without that feature flag, and not implementing `zeroize` on their field impl, and it works fine for downstream users until some other crate enables `zeroize`, causing compilation errors. This is "spooky breakage at a distance", and often unsolvable by the downstream user.

The way around this is to move the bound into a separate trait that is behind the feature flag (so it becomes an API expansion: downstream libraries that don't implement the new trait won't break when the feature flag is enabled). But we already have this with `zeroize::Zeroize`, so there is no need to do anything in this crate.

Reply:
Ah, I see. I'll fixup this change accordingly then.